tests/test_pricer.py

```diff
@@ -10,6 +10,7 @@ def pricerredcost(self):
         dualSolutions = []
         for i, c in enumerate(self.data['cons']):
             dualSolutions.append(self.model.getDualsolLinear(c))
+            #assert self.model.getDualsolLinear(c) == self.model.getDualSolVal(c)
```

Comment:
Why is this commented out?

Reply:
I forgot to delete it, but the assertion fails, ```getDualSolVal``` returns 0, even with ```boundconstraint=True```. 

As a side not, it's annoying that sol is not capitalized in ```getDualsolLinear````

Reply:
```suggestion
```
Let's remove the line then for now. The naming comes from the corresponding SCIP C function, better to be consistent even if not the best naming.

Reply:
I opened a new issue #746  to track this.